articles/logs/streams/datadog.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Datadog Event Log Streams
+description: Datadog Event Log Streams let you export your events in near real-time to Datadog.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# Datadog Event Log Streams
+
+Datadog Event Log Streams let you export your log events to Datadog for further analysis and monitoring. When Auth0 creates a log entry for your tenant, a copy is automatically sent to Datadog with the `source` and `service` both set to `auth0`.
```

Comment:
For consistency with the eventbridge docs I think this paragraph should give an overview of the value of the integration. 

Datadog is the essential monitoring platform for cloud applications. That brings together together data from servers, containers, databases, and third-party services to make your stack entirely observable.

You can create monitoring, alerting and analysis dashboards for Auth0 tenants.

Reply:
Added paragraph about Datadog value